zingo-testutils/src/chain_generic_tests.rs

```diff
@@ -131,33 +135,39 @@ pub mod fixtures {
     }
 
     /// sends back and forth several times, including sends to transparent
-    pub async fn send_shield_cycle<CC>(n: u64)
+    pub async fn send_shield_cycle<CC>(n: u64, primary_pool: PoolType)
```

Comment:
I don't understand, this looks like a test but has no asserts? can you explain the context of this function?

Reply:
the unwraps function as assertions in this test. but more assertions are coming